sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/tests/SchemaRegistryAvroObjectSerializerLiveTest.cs

```diff
@@ -69,6 +71,7 @@ public async Task CanSerializeAndDeserializeGenericRecord()
         }
 
         [Test]
+        [PlaybackOnly("Service not provisioned to all regions yet.")]
```

Comment:
Any reason to not set these at the class level?

Reply:
Didn't think about that. 🙃 Since these attributes are temporary, it hopefully shouldn't make any fundamental difference.